micrometer-core/src/test/java/io/micrometer/core/instrument/binder/httpcomponents/MicrometerHttpRequestExecutorTest.java

```diff
@@ -39,7 +39,7 @@
 import java.util.stream.Collectors;
 
 import static com.github.tomakehurst.wiremock.client.WireMock.*;
-import static org.assertj.core.api.Java6Assertions.assertThat;
```

Comment:
I wonder if we shouldn't ban this import. Is there a reason for us being Java 8 base to ever use the Java6Assertions package?

Reply:
@shakuzen Not that I know of. I created https://github.com/micrometer-metrics/micrometer/pull/1731.